src/Build/mkdocs.jl

```diff
@@ -0,0 +1,94 @@
+function makemkdocs(build)
+    site_name = "Docile.jl"
+    repo_url = "https://github.com/MichaelHatherly/Docile.jl"
+    site_description = "Julia package documentation system."
+    site_author = "Michael Hatherly"
+    theme = "readthedocs"
+    docs_dir = joinpath(splitdir(Base.source_dir())[end], build)
```

Comment:
Probably best to not require network connection to build docs I guess. Though we could have it as an option if available.